sentinel-2/ozone_column_over_snow/README.md

```diff
@@ -0,0 +1,39 @@
+- [EO Browser](https://sentinelshare.page.link/cFC7){:target="\_blank"}
+
+## General description of the script
+
+Kokhanovsky et al. 2021 propose a simple algorithm to derive the total ozone column and snow properties (spectral albedo and effective light absorption path) from Sentinel-2 L1C measurements over Antarctica. This script is an implementation of the algorithm that calculates the Total Ozone Column over highly-reflective surfaces located beneath a clean Antarctic atmosphere. The algorithm uses reflectance in the spectral range 443–865 nm, more particularly the maximal ozone absorption band (559.8 nm), the maximal ice absorption band (864.7 nm) and the minimum ice and ozone absorption band (442.7 nm), to calculate the K parameter which is correlated to the total ozone column.
```

Comment:
According to a spell check you can put a comma before "and":  "... ice absorption band (864.7 nm), and the minimum ice ..."